test/tls.uts

```diff
@@ -984,10 +1064,32 @@ t.len = 0xefff
 str(t) == b'\xde\xad\xbe\xef\xff\x01'
 
 
+= Building packets - TLS record with bad data
+a = TLS(b'\x00\x03\x03\x00\x03data')
+assert a.haslayer(Raw)
+
+
+= Building packets - _CipherSuitesField with no cipher
+from scapy.layers.tls.handshake import _CipherSuitesField
+a = _CipherSuitesField("test", None, {})
+assert a.i2repr(None, None) == "None"
+assert isinstance(a.randval(), RandBin)
+assert hash(a) == 8890308
```

Comment:
Meh, now you ask it I should have added a ˋrandom.seed` before it to be surévalué it's the same...
You can either add that call, either remove it. Up to you